datajoint/table.py

```diff
@@ -170,7 +170,7 @@ def insert(self, rows, replace=False, skip_duplicates=False, ignore_extra_fields
         # prohibit direct inserts into auto-populated tables
         if not (allow_direct_insert or getattr(self, '_allow_insert', True)):  # _allow_insert is only present in AutoPopulate
             raise DataJointError(
-                'Auto-populate tables can only be inserted into from their make methods during populate calls.')
+                'Auto-populate tables can only be inserted into from their make methods during populate calls. (see allow_direct_insert)')
```

Comment:
```suggestion
                'Auto-populate tables refuse direct inserts from outside their make methods during populate calls, unless overridden with the allow_direct_insert argument.')
```

Reply:
Hmm.. I see combining the two, but think the original 'error' description to me seems more straightforward- what do you think about:

```
Auto-populate tables can only be inserted into from their make methods during populate calls, unless overridden with the allow_direct_insert argument.```

or

```Auto-populate tables can only be inserted into from their make methods during populate calls. To override, use the the allow_direct_insert argument.```

was thinking something like this at 1st but deliberately was a bit obscure so as to not 'encourage' bad behavior by users looking for a 'quick fix' to the issue..

will plan to go with your choice of any from here.

Reply:
Agreed.
```suggestion
                'Auto-populate tables can only be inserted into from their make methods during populate calls.' \
                ' To override, use the the allow_direct_insert argument.')
```